mysql/literal.go

```diff
@@ -55,6 +57,17 @@ var String = jet.String
 // value can be any uuid type with a String method
 var UUID = jet.UUID
 
+// UUIDToBin takes ay object with a String method and calls StringUUIDToBin.
+func UUIDToBin(str fmt.Stringer) StringExpression {
```

Comment:
* Naming convention is to use all uppercase for all sql functions, so the name should be `UUID_TO_BIN`
* `uuid_to_bin` accepts not only parameters, but also columns and expression, so function parameter should be `StringExpression`
* `functions.go` is probably more appropriate place.

Reply:
@go-jet makes sense. This also eliminates the need for the `StringUUIDToBin` function.